cmake/extensions.cmake

```diff
@@ -2092,23 +2094,42 @@ endfunction()
 #                                  BUILD debug, will look for <board>_debug.conf
 #                                  and <board>_debug.overlay, instead of <board>.conf
 #
-# returns an updated list of absolute paths
+# FILENAMES <list>: Construct a filename based on the provided BOARD, REVISION, and BUILD type.
```

Comment:
Returns a list of filenames based on the BOARD, REVISION, and BUILD type.

Reply:
done